google-cloud-firestore/samples/add_data.rb

```diff
@@ -148,6 +148,32 @@ def update_doc project_id:, collection_path: "cities"
   puts "Updated the capital field of the DC document in the cities collection."
 end
 
+def update_doc_array project_id:, collection_path: "cities"
+  # project_id = "Your Google Cloud Project ID"
+  # collection_path = "cities"
+
+  firestore = Google::Cloud::Firestore.new project_id: project_id
+
+  doc_ref = firestore.doc "#{collection_path}/DC"
+
+  data = {
+    name:    "Washington D.C.",
+    country: "USA",
+    regions: ["east_coast"]
+  }
+  doc_ref.set data
+  # [START firestore_data_set_array_operations]
```

Comment:
This tag excludes the construction of `firestore` as well as the setting of `project_id` and `collection_path`. It needs to include those lines above so that users can copy-paste a working sample. If we want to exclude the `doc_ref` setting, we can temporarily END and then re-START in the middle.

Reply:
The Firestore samples have been scoped like this [since the very beginning](https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/275/files). On https://firebase.google.com/docs/firestore/manage-data/add-data if you tab through the languages, they all use this "abridged" style.

Reply:
Ugh. I see. Okay, well, I guess we should follow precedent for now, but I may raise this internally and try to revisit it across languages.